specification/schemas/returns/ReturnSimpleResponse.json

```diff
@@ -47,6 +47,56 @@
               "example": "$API_HOST/returns/v1/returns/8ddfb83f-d600-49c7-b10e-66dd4dc7686a"
             }
           }
+        },
+        "meta": {
+          "readOnly": true,
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "shipment": {
+              "type": "object",
+              "additionalProperties": false,
+              "required": [
+                "tracking_code",
```

Comment:
If the shipment is registration failed, there is a status but no tracking code
```suggestion
```

Reply:
Should we also expose the `tracking_url`? I assume when showing a tracking code somewhere that we want to make it link to the tracking website.